modules/host/negotiate.go

```diff
@@ -75,7 +75,7 @@ func (h *Host) considerTerms(terms modules.ContractTerms) error {
 	case len(terms.MissedProofOutputs) != 1:
 		return errors.New("refund len does not match host settings")
 
-	case terms.MissedProofOutputs[0].UnlockHash != types.ZeroUnlockHash:
+	case terms.MissedProofOutputs[0].UnlockHash != types.UnlockHash{}:
```

Comment:
honestly it felt clunky to me, especially since you actually lose characters. I also do not feel that readability was lost, it seems more clear to me that UnlockHash{} is an empty structure than `ZeroUnlockHash` (not that either is particularly opaque).

Mostly it just felt like an unnecessary alias, and I think you should try to eliminate everything in your codebase that doesn't directly contribute a lot of value.